browser/components/ContributionEditor.tsx

```diff
@@ -304,7 +305,8 @@ class ContributionsEditor extends React.Component<Partial<Props>, State> {
                 <textarea className="form-control" rows={2} id="approvalNotesInput" placeholder={approval_notes} />
               </div>
               <div className="form-group">
-                <button className="btn btn-primary" type="submit">Submit</button>
+                <Link className="btn btn-secondary im-blue" id="to-admin" to="/admin">Cancel</Link>
```

Comment:
Cancel buttons make sense on modals and things where the previous page has "state", but here (and some other places) it just directs to some other page. Is there another use for this?

Reply:
This is because the admin section is only a single that switches what is displayed based on context. I direct to the admin page so that it lands at the start of that work flow and has the most recent data(if someone updated something under you).